src/System.Collections.Concurrent/tests/Performance/ConcurrentQueuePerfTest.netcoreapp.cs

```diff
@@ -0,0 +1,44 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Xunit.Performance;
+using Xunit;
+
+namespace System.Collections.Concurrent.Tests
+{
+    public class Perf_ConcurrentQueueTests
+    {
+        /// <summary>
+        /// Creates a list containing a number of elements equal to the specified size
+        /// </summary>
+        [Benchmark]
+        [InlineData(8, 1000)]
+        [InlineData(50, 1000)]
+        public void Enqueue_Dequeue(int threadsCount, int itemsPerThread)
+        {
```

Comment:
Most of the [Benchmark] tests are looping over iterations similarly to the example here: https://github.com/dotnet/corefx/blob/master/Documentation/project-docs/performance-tests.md

Probably need to do that here along with calling iteration.StartMeasurement().